examples/transaction_example_as_library_master.py

```diff
@@ -0,0 +1,35 @@
+import gevent
+import locust
+from locust import runners
+from locust.main import create_environment
+from locust.env import Environment
+from examples import transaction_example
+from locust import HttpUser, task, SequentialTaskSet
+from locust_plugins.transaction_manager import TransactionManager
+
+class ExampleSequentialTaskSet(SequentialTaskSet):
+    def on_start(self):
+        TransactionManager().on_locust_init(env, env.runner)
+        self.tm = TransactionManager()
```

Comment:
This looks weird. You’re creating a new TM on both lines?

Reply:
oops, I should have checked this more carefully.